recipes-soletta/soletta/soletta_0.1.bb

```diff
@@ -7,10 +7,10 @@ SECTION = "examples"
 DEPENDS = "glib-2.0 libpcre python3-jsonschema"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://COPYING;md5=53eeaddf328b23e2355816e257450eaa"
-PV = "0.0+git${SRCPV}"
+PV = "1_beta2"
 SRCREV = "${AUTOREV}"
```

Comment:
Since it is now a fixed version. It is needed to remove SRCREV.

 When tested locally I had some conflict and yocto aborted the building saying that I have to choose between AUTOREV or the PV version